src/sqllogictest/src/bin/sqllogictest.rs

```diff
@@ -147,6 +147,9 @@ struct Args {
     /// Wrapper program to start child processes
     #[clap(long, env = "ORCHESTRATOR_PROCESS_WRAPPER")]
     orchestrator_process_wrapper: Option<String>,
+    /// Number of replicas, defaults to 2
+    #[clap(long, default_value = "2")]
```

Comment:
I think the default needs to stay 1 here and then be set to "2" explicitly in buildkite. This way the current behavior will remain unchanged.

You can also consider not modifying sqllogictest.rs at all, but rather create the correct number of replicas in the mzcompose.py file. This will bring SLT+replicas in line with testdrive and the other frameworks, where any replica shenanigans are done in mzcompose.py code -- the rust part of testdrive is not creating any replicas for example.

Reply:
> You can also consider not modifying sqllogictest.rs at all, but rather create the correct number of replicas in the mzcompose.py file. This will bring SLT+replicas in line with testdrive and the other frameworks, where any replica shenanigans are done in mzcompose.py code -- the rust part of testdrive is not creating any replicas for example.

Actually disregard this, I spaced out. SLT is running an embedded Mz instance, so it needs to do its own replica management, it can not be offloaded to mzcompose.